x-pack/plugins/security/server/authentication/authentication_service.ts

```diff
@@ -187,55 +187,71 @@ export class AuthenticationService {
     });
 
     http.registerOnPreResponse(async (request, preResponse, toolkit) => {
-      if (preResponse.statusCode !== 401 || !canRedirectRequest(request)) {
-        return toolkit.next();
-      }
-
       if (!this.authenticator) {
         // Core doesn't allow returning error here.
         this.logger.error('Authentication sub-system is not fully initialized yet.');
         return toolkit.next();
       }
-
```

Comment:
__optional nit:__ a bit weird that we removed a new line separating comment from the unrelated code block (`!this.authenticator`), but added a new line separating the comment from the code block it's actually related to (`const originalURL `) :slightly_smiling_face: